test/celeritas/CMakeLists.txt

```diff
@@ -259,6 +259,10 @@ celeritas_add_test(field/LinearPropagator.test.cc
   LINK_LIBRARIES ${_all_geo_libs})
 celeritas_add_test(field/MagFieldEquation.test.cc)
 
+if(CELERITAS_USE_Covfie)
+  celeritas_add_test(field/CartField.test.cc)
+endif()
```

Comment:
Ideally, we would have all the `#if COVFIE` stuff inside the CartMapField classes, and this could just live in the `Fields.test.cc` with 
```
#if !CELERITAS_USE_COVFIE
#define CartFieldTest CylMapFieldTest_DISABLED
#endif
```
... this would require hiding the data includes, and putting a null-op implementation in the field if covfie isn't enabled. I know it sounds like a bit of work but it localizes all the implementation details to that one part of the codebase. In the future anything that uses the cartesian field wouldn't have to change if we switched out the back end.